beacon-chain/blockchain/van_process_pending_blocks.go

```diff
@@ -241,7 +273,7 @@ func (s *Service) verifyPandoraShardInfo(parentBlk, curBlk interfaces.SignedBeac
 			WithField("canonicalShardingBlkNum", canonicalShardingBlkNum).WithField("curShardingParentHash", curShardingParentHash).
 			WithField("curShardingBlockNumber", curShardingBlockNumber)
 
-		if curShardingParentHash != canonicalShardingHash && curShardingBlockNumber != canonicalShardingBlkNum+1 {
+		if curShardingParentHash != canonicalShardingHash || curShardingBlockNumber != canonicalShardingBlkNum+1 {
```

Comment:
Nice catch.

Reply:
Thank you, but this is because of @blazejkrzak :+1: